frontend/src/pages/BYONImages/BYONImageModal/DisplayedContentTableRow.tsx

```diff
@@ -60,6 +60,7 @@ const DisplayedContentTableRow: React.FC<DisplayedContentTableRowProps> = ({
           {isActive ? (
             <TextInput
               aria-label={`${dataLabel} name input`}
+              data-testid={`${dataLabel} name input`}
```

Comment:
@christianvogt Adding a test ID works fine for a single row in both the 'packages' and 'software' tabs
when dealing with multiple rows in these tabs, how to select a specific row since each row might contain either data or text input???